runtime/litentry/src/lib.rs

```diff
@@ -123,6 +124,12 @@ pub type Executive = frame_executive::Executive<
 	Block,
 	frame_system::ChainContext<Runtime>,
 	Runtime,
+	// see https://github.com/paritytech/substrate/pull/10043
+	//
+	// With this type the hooks of pallets will be executed
+	// in the order that they are declared in `constrcut_runtime!`.
+	// It was reverse order before.
+	// See the comment before collation related pallets too.
 	AllPalletsWithSystem,
```

Comment:
See #336 
I think the safest move for now is stay with AllPalletsWithSystemReversed

Reply:
> See #336 I think the safest move for now is stay with AllPalletsWithSystemReversed. I also feel it should do no harm to us though. Let's discuss then decide.

I was a bit undecided, but after I saw the discussion in the polkadot companion, I'm inclined to have the new `AllPalletsWithSystem` (+ extra confirmation from Basti)

Reply:
As discussed, I'll put a comment in code later that we have to pay attention to the order of inherent calls.